Library/Loop.cs

```diff
@@ -18,6 +18,9 @@ namespace BMeshLib
      * namely the radial list, that enables iterating over all the faces using
      * the same edge.
      */
+    /// <summary>
+    /// Represents a portion of a <see cref="Face"/>.
```

Comment:
All above comment again :) What is a bit misleading is that a `Loop` object can be seen either as representing the whole loop or one node (i.e. element) of this loop. It's a bit like with a linked list:

```C#
class LinkedList {
    object head;
    LinkedList tail;
}
```

Here a `LinkedList` technically only contains the data of the head, so only one of the nodes of the list, but it can be thought as the whole list as well. It's more subtle with the Loop because first it's a linked list with the end connected to the beginning, so that there is no item flagged as "end" and "beginning" actually, any item can represent the whole loop; and secondly it is also used as an element/node of another looped list, which contain all the edges which share a same vertex.

Maybe the most intuitive way to describe a loop item is to tell it is the combination of a vertex, an edge and a face, and that it is meant to provide fast access to neighboring edges, either by "turning around" the face (Next/Prev) or "turning around" the vertex (RadialNext/RadialPrev).

(BTW I realized lately that a face can be made of several sequences of loop actually, say if it contains a hole in the middle, but this is an edge case we don't support here)

Reply:
To be honest, I feel like the technical comment doesn't actually clear anything up, at least for me. Though I also don't like the doc comment I write since it doesn't really tell anything useful about the class itself.

While writing the doc comments, I am trying to get away from telling the user how the underlining structure works, and instead tell them how it works in the context of the rest of the API. While my doc comment for the Loop isn't great, I feel like it at least illustrates what I mean. It tells you what the Loop is in relation to the rest of the API.

> Maybe the most intuitive way to describe a loop item is to tell it is the combination of a vertex, an edge...
I think that this is the most understandable way I have seen to explain a Loop.

Reply:
> "turning around" the vertex (RadialNext/RadialPrev).

What are you talking about here?

Reply:
![image](https://user-images.githubusercontent.com/5802849/132248424-eb20ea2a-27dc-4065-b2b3-0a3482b00d90.png)

Reply:
If you don't understand the technical comment don't include it to <summary> but please don't remove it it is at least useful for myself to remember

Reply:
I need to look at it more, but I think that diagram helps. Of course! I didn't remove any of the technical comments on the classes, I think they are important when working with the source code to describe how and why things are structured!